### ISSUE TITLE
Links have the same name

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#578]
Issue group: Tables
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 2.4.4: Link Purpose (In Context)
Needs design: 

### Description
Generally, link text is clear or can be determined from context, with a few exceptions. 

E.g., On the API keys page, a table contains several links which all have the same name (“Revoke”).

E.g., the table on the Settings page contains several links that all have the same link text (Change). 

Some context is provided through other cells within the table row, however this does not strictly meet the relevant [WCAG Technique H79](https://www.w3.org/WAI/WCAG21/Techniques/html/H79)). The purpose of the link should be clarified through the table cell content, or the table header. Alternatively, a different technique should be used to clarify the purpose of the link. Consider one of the following approaches: 
1. Make the column that clarifies what the user will be revoking, changing, etc. a row header cell (<th scope=”row”)
2. Add an aria-describedby attribute to each link that will associate it with the related cell.

https://www.w3.org/WAI/WCAG21/Techniques/html/H79

![](https://dl.airtable.com/.attachments/4a916cbb822a2bd04ef9293aa2d9dbdb/cb55e930/Capturedecranle2022-03-15a13_33_58.png, https://dl.airtable.com/.attachments/f30053030525c076296499f3697150d5/985e203f/Capturedecranle2022-03-15a13_34_25.png)

**Comment by yaelberger-commits** [comment]
Some of this issue might be fixed when we did some updates to tables

**Comment by yaelberger-commits** [comment]
Next step is to validate if we fixed this in both places, if yes, close this card, if not completely done, can pick it up in Sprint 7

**Comment by whabanks** [comment]
Changes have been made to the services settings page to clarify what it is the user is changing when they land on any of the change links.

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/578?planning-poker) with Zenhub @whabanks

**Comment by adriannelee** [comment]
QA'd! LGTM

**Comment by adriannelee** [comment]
QA'd! LGTM
